composer.json

```diff
@@ -49,7 +49,7 @@
     },
     "config": {
         "platform": {
-            "php": "7.1"
+            "php": "7.1.19"
```

Comment:
Why do we need this?

This is a real bummer, e.g. because I'm still on earlier version at places, where it is very cumbersome to upgrade.

Reply:
We need at least 7.1.3 for symfony 4

Reply:
I'm fine with 7.1.3 if you ask, but I think we'll be fine with just `7.1` in this field.

Reply:
It wouldnt update symfony for me with 7.1 in the config. Ill update it to 7.1.3